src/components/filters/filter-dynamic/filter-dynamic.component.less

```diff
@@ -22,6 +22,10 @@ ux-filter-dynamic {
             font-weight: 600;
         }
 
+        .filter-title {
+            text-transform: capitalize;
```

Comment:
I believe it should be the header name (which you have used the `group` property to represent) that should be transformed, per this screenshot:
![image](https://user-images.githubusercontent.com/26114101/76447392-82f22780-63c0-11ea-8591-158499b7939a.png)

You would therefore need to apply (and document) a class for that part of the button to provide the correct styling.

Also, you are probably looking for `text-transform: uppercase` rather than `capitalize` when implementing that class.

Reply:
Hi Donna thanks for the feedback on this. 

The reason I have used the capitalise instead of uppercase was I understood the invision concepts Roland provided was for the micro focus theme and that the Keppel styling should remain as is (albeit with some minor changes to accommodate this new functionality). 

The uppercase styling is implemented into the micro focus theme in the relevant quantum and quantum-ux-aspects style sheets.

Apologies if I have misunderstood!

Reply:
That's fine, if you are looking to leave the text unchanged you can just leave this out because `text-transform: none` has already been specified:
![image](https://user-images.githubusercontent.com/26114101/76615262-9664d600-6519-11ea-8d42-b2b41cf51ff7.png)